File: async-scala-client/src/main/scala/io/swagger/client/model/Payment_bill.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Payment_bill (
  nFat: Option[String],  // Bill identifier
vOrig: Option[Double],  // Orignal value
vDiscount: Option[Double],  // Discount
vNet: Option[Double]  // Net value
)

File: javascript-closure-angular-client/API/Client/PaymentBill.js
goog.provide('API.Client.Payment_bill');

/**
 * Bill information
 * @record
 */
API.Client.PaymentBill = function() {}

/**
 * Bill identifier
 * @type {!string}
 * @export
 */
API.Client.PaymentBill.prototype.nFat;

/**
 * Orignal value
 * @type {!number}
 * @export
 */
API.Client.PaymentBill.prototype.vOrig;

/**
 * Discount
 * @type {!number}
 * @export
 */
API.Client.PaymentBill.prototype.vDiscount;

/**
 * Net value
 * @type {!number}
 * @export
 */
API.Client.PaymentBill.prototype.vNet;

